src/main/scala/Main.scala

```diff
@@ -88,27 +90,39 @@ object Main extends App {
         .map(c => c.sortKey)
     }
 
-    val wikiModel = new WikiModel("wiki/${image}", "wiki/${title}")
+    // not unused for now
+    // val wikiModel = new WikiModel("wiki/${image}", "wiki/${title}")
+
     val sortedEntries = labels.keys.toArray.sortWith(_.length > _.length)
-    val sentenceOption = SentenceOption("。", "", maxTokens)
+    val sentenceOption = SentenceOption("。", "", maxTokens, minTokens)
     val annotationOption = AnnotateOption("")
     val outputWriter = new FileWriter(new File("annotations.txt"))
+    val printEvery = 500
     var count = 0
+    var annotationList = List[String]()
 
     while (pageQueue.nonEmpty) {
       val dq = pageQueue.dequeue
       val link = dq._1
       pageQueue = dq._2
       val source = DbHelper.getPageSource(conn, link.pageId)
       if (source.nonEmpty) {
-        val text = wikiModel.render(new PlainTextConverter(), source.get)
+        val text = source.get
         val sentences = WikiTextHelper.sentencesContains(text, link.title, sentenceOption)
         val annotations = sentences.map(WikiTextHelper.annotate(_, sortedEntries, labels, annotationOption))
-        annotations.foreach({ s => outputWriter.write(s"$s\n")})
-        count += annotations.size
-        println(s"Total annotations: $count")
+        annotationList = annotationList ++ annotations
       }
     }
+
+    // Shuffle the examples
+    annotationList = Random.shuffle(annotationList)
+    annotationList.foreach({ s =>
+      count +=1
+      if (count % printEvery == 0) {
+        println(s)
```

Comment:
Just logging the output every 500 lines to take a glance of what was actually written into the output.